types/rollup-plugin-less/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for rollup-plugin-less 1.1
+// Project: https://github.com/xiaofuzi/rollup-plugin-less##readme
+// Definitions by: Tristan <https://github.com/LeoDog896>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="node" />
+/// <reference types="jsdom" />
```

Comment:
```suggestion
```
do you need this?

Reply:
It didn't seem to work without it -- I'll test it locally first.

Reply:
no worries, that can be required, let me know

Reply:
Completely forgot about this and came back -- it seems that less requires HTMLLinkElement which is why I added jsdom.